roles/tidb/vars/default.yml

```diff
@@ -179,7 +179,3 @@ pessimistic_txn:
 
   # max retry count for a statement in a pessimistic transaction.
   max-retry-count: 256
-
-  # default TTL in milliseconds for pessimistic lock.
-  # The value must between "15s" and "60s".
-  ttl: "30s"
```

Comment:
LGTM, but we have to make sure rolling updates with the old configures also work.

Reply:
link https://github.com/pingcap/tidb/pull/13142